code/api/src/modules/user/resolvers.js

```diff
@@ -58,6 +58,20 @@ export async function login(parentValue, { email, password }) {
   }
 }
 
+// Update user
+export async function update(parentValue, { id, email }, { auth }) {
+  if (auth.user && auth.user.role === params.user.roles.user || params.user.roles.admin) {
```

Comment:
May have to change this around a bit in the future for security purposes, but for now it's fine I think

Reply:
Yeah, I am not sure how to verify the specifically authenticate a particular user.